### DISCUSSION TITLE
Welcome to joplin-sheets Discussions!

### DISCUSSION THREAD

**Discussion by ThibaultJanBeyer** [ThibaultJanBeyer/joplin-sheets#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by W3S-L33** [comment]
Hi @ThibaultJanBeyer ! I'm a new Joplin user and your plugin sold the package to me. I'm liking Joplin so far but was lacking a proper spreadsheet. Seems like you built a solid plugin. Thanks for your effort!
